src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Typography } from '@components/atomic';
-import { elideAddress } from '@utilities';
+import { doNothing, elideAddress } from '@utilities';
```

Comment:
The idea is quite nice. Imagine you have a React component where you are expected to receive a prop which is function ( a callback ). For example:

```
const A = ({ cb }) => <div onClick={() => cb('123')} />
```

now if someone uses this component and doesn't pass cb this code will break...

you can easily avoid this by following a param init pattern, in case the callback is not provided in the following manner:


```
const A = ({ cb = doNothing }) => <div onClick={() => cb('123')} />
```

now even if someone doesn't pass cb, the cb will default to doNothing which won't break the code due to bug deployed to prod.

Reply:
oh, that neat!